### DISCUSSION TITLE
posts/2023/10/steadfast-feeling/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#25] [General]
# posts/2023/10/steadfast-feeling/

https://guozheng.rbind.io/posts/2023/10/steadfast-feeling/

**Comment by CyrusYip** [comment]
你有没有去看过皮肤科？以前我也不懂为啥要洗被套晒被子，还是皮肤科医生教我的。我感觉自己从医生那里学了不少良好的生活习惯。推荐你用丁香医生查医学知识，搜「螨虫」你就能看到防治方法，它的问诊服务也很棒，医生回应很详细。

药箱是月饼盒吧，哈哈。

**Reply by residualsun1** [reply]
> 你有没有去看过皮肤科？

还没有。我对大医院的信任度很低，大一的时候去检查眼底，医生建议我动手术，后来家里人把我的检查结果给认识的同样在医疗系统的朋友看了一下，说注意休息就好。前段时间治手指甲的时候，医院的医生做了几个检查就给我开会伤肝的药，远不如我们家找的专业机构（比如郑远元在全国连锁，专治指甲）。不过现在脸部也还好，就是晚上稍微会有点痒，但是没有什么明显的生理症状，我打算把生活习惯矫正好，再观察一下。

丁香医生我现在也在用，好像是从小红书上搜治螨虫的信息链接到的。我感觉生活中有一些很细碎的细节可以参考一下小红书，不过我搜完也删除了（不大喜欢里面的社区生态，贴吧、微博、抖音我也全都不用）。但像丁香医生这种专业性比较强的软件我会留着。

> 药箱是月饼盒吧，哈哈。

每年中秋我妈都要给我寄月饼，不过我确实不喜欢吃蛋黄，也不喜欢金九，基本都分给了舍友，所以最后对我来说真正有需要的是可以收纳东西的月饼盒哈哈。

**Reply by CyrusYip** [reply]
@residualsun1  我以前经常看病，确实有时候看不好或者对治疗方案不满意，可能是医院/医生水平有限，甚至是医学发展水平有限。对于慢性病，我的经验是习惯带病生存，不放弃治疗，但也别太期待治愈。我似乎没听过医生说「治愈」这个词语，只是听到「治疗」，医学还是有很大进步空间。生病固然痛苦，但是太纠结又多了额外的痛苦。

我在好几个地方看到说在小红书搜生活知识比谷歌好使，一直纠结要不要注册。我觉得国内的平台太封闭了，看个帖子都要安装手机应和手机号注册，所以基本不用，甚至还注销了。

**Reply by residualsun1** [reply]
> 对于慢性病，我的经验是习惯带病生存，不放弃治疗，但也别太期待治愈。

唉，偶尔会有过痊愈的期待，不过最终也不知不觉变成了习惯。

> 我在好几个地方看到说在小红书搜生活知识比谷歌好使，一直纠结要不要注册。

这个就看需求了。不过我觉得它不是非常必需哈哈，只是可能在刚好能解答你的某个时刻才会让你觉得眼前一亮，但大多时候还是用不上的。所以我往往是下载了，没两天就又删除了——我对手机的简洁很执着，能精简就精简。
